scripts/bootstrap.js

```diff
@@ -35,7 +35,7 @@ ls("packages/*").forEach(function (loc) {
 
 var completed = false;
 
-var bar = new ProgressBar(":packagename ╢:bar╟", {
+var bar = !/^win/.test(process.platform) && new ProgressBar(":packagename ╢:bar╟", {
```

Comment:
The bootstrap script has been moved into a separate project called [Lerna](https://github.com/sebmck/lerna) so this file will need to be fixed separately over there.